src/lib.rs

```diff
@@ -73,13 +74,26 @@ pub enum Verbosity {
 impl Verbosity {
     /// Get the verbosity level from the `RUST_BACKTRACE` env variable.
     pub fn from_env() -> Self {
-        match std::env::var("RUST_BACKTRACE") {
-            Ok(ref x) if x == "full" => Verbosity::Full,
-            Ok(_) => Verbosity::Medium,
-            Err(_) => Verbosity::Minimal,
+        Self::convert_env(env::var("RUST_BACKTRACE").ok())
```

Comment:
Should these methods follow std's example and make some effort to cache these values?

Reply:
Nah, I think this is fine as-is. I'd argue that panic printers and thus verbosity instances are generally constructed only few times per app-run and it wouldn't be worth the effort.